### ISSUE TITLE
[Create event] When we go to 'Initiative type' with the 'Tab' key, we cannot select with the 'Enter' key

### ISSUE THREAD

**Issue by Tetiana1202** [ita-social-projects/GreenCity#6825]
Environment: MacOS 13.5.1, Google Chrome (120.0.6099.129) .
Reproducible: always
Build found: 08.01.2024

Preconditions:
 1. Go to https://www.greencity.social/#/greenCity
 2. User is Authorized
 3. Open "Events" page

Steps to reproduce:
1. Click on  'Create Event' button
2. Use the "Tab" key to go to "Initiative type", namely to the "Economic" button.
3. Press the 'Enter' key
5. Use the "Tab" key to go to the "Social" button
6. Press the 'Enter' key
7. Use the "Tab" key to go to the "Environmental" button
8. Press the 'Enter' key

Actual result:
1. The "Economic", "Social", "Environmental" buttons are not active

Expected result:
1.  "Economic", "Social", "Environmental" buttons are active, the buttons are displayed in green.
